qa/L0_inferentia_perf_analyzer/test.sh

```diff
@@ -61,10 +61,8 @@ source /opt/tritonserver/qa/common/util.sh
 TEST_TYPES="single multiple"
 BATCHED_FLAGS="_ _batched_"
 DISABLE_DEFAULT_BATCHING_FLAGS="_default_batch _no_batch"
-
-# Setup models
-if [ ${USE_TENSORFLOW} == "1" ]; then
-    TEST_FRAMEWORK="tf1"
+# Helper function for generating models
```

Comment:
the clear_model_dir does not generate models, move to above create_inferentia_models